<launch>
  <!-- Elevation mapping node -->

  <include file="$(find subt_exploration)/launch/filter_points.launch">
    <arg name="points_window_size" value="6.0"/>
  </include>

  <node pkg="elevation_mapping" type="elevation_mapping" name="elevation_mapping" output="screen">
    <rosparam command="load" file="$(find subt_exploration)/config/mapping/elevation_config.yaml" />
    <!-- <rosparam command="load" file="$(find atwifr_bringup)/param/elevation_config/map.yaml" /> -->
    <rosparam command="load" file="$(find elevation_mapping_demos)/config/postprocessing/postprocessor_pipeline.yaml" />
  </node>

  <!-- Setup a transform between the world and the robot -->
  <node pkg="tf" type="static_transform_publisher" name="world_to_robot" args="0.0 0.0 0 0 0.0 0 /map /COSTAR_HUSKY/odom 100"/>

  <!-- Launch visualizations for the resulting elevation map -->
  <include file="$(find elevation_mapping_demos)/launch/visualization.launch" />

  <!-- Launch RViz with the demo configuration -->
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find elevation_mapping_demos)/rviz/elevation_map_visualization.rviz" /> -->

</launch>